### ISSUE TITLE
Allow reverse proxy to be enabled only for certain features

### ISSUE THREAD

**Issue by jordienr** [PostHog/posthog#16880]
## Is your feature request related to a problem?
We are using the reverse proxy feature with Nextjs/Vercel so adblocks don't block our feature flag checks. The issue is every event is sent through the proxy, if a user is using an adblock we want to respect that and only use the proxy for the feature flag checks.

## Describe the solution you'd like
Be able to pass a host_url param to the feature flags in posthog.init()
```typescript
posthog.init(key, {
        feature_flags: {
            api_host: 'myproxy.com'
        },
    });
```

**Comment by empz** [comment]
Yes, please, this is a must. We need feature flags to work when users are blocking trackers or don't want to use reverse proxy for session recording data.

**Comment by empz** [comment]
No updates on this one? This looks like a very reasonable request. I can't use client-side feature flags because a lot of people have tracking blocked and I respect that. Also I don't want to proxy replay data as it can consume a lot of bandwidth.

We need a simple way to proxy feature flag requests.
